CSS/style.css

```diff
@@ -0,0 +1,103 @@
+@import url("/CSS/utils.css");
+
+.logo-link {
+  font-size: 1.5rem;
+}
+.rules-link {
+  font-size: 1rem;
+}
+.rules-link:hover,
+.profile-icon:hover {
+  color: var(--Primary_Color);
+}
+
+/* Intro section */
+.app-intro {
+  padding: 1rem;
+  display: flex;
+  gap: 10px;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+.intro-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.intro-header,
+.intro-quote,
+.intro-greeting {
+  font-weight: 400;
+  opacity: 0.9;
+  margin: 0;
+  font-size: 1.5rem;
```

Comment:
Maybe you can make utility classes for font sizes that will be very helpful in further projects as well.

Reply:
I will do that, thanks.